src/game/common/rts/player.cpp

```diff
@@ -47,11 +47,13 @@ void Player::Xfer_Snapshot(Xfer *xfer)
 }
 
 // zh: 0x00452E40, wb: 0x0085C643
-void Player::Compute_Superweapon_Target(const SpecialPowerTemplate *sp_template, Coord3D *loc, int32_t unk1, float unk2)
+bool Player::Compute_Superweapon_Target(const SpecialPowerTemplate *sp_template, Coord3D *loc, int32_t unk1, float unk2)
```

Comment:
In assembler it looks like this returns a byte. So I changed to bool here for now.